api4/status.go

```
@@ -71,6 +71,11 @@ func updateUserStatus(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	currentStatus, err := c.App.GetStatus(c.Params.UserId)
```

We shouldn't ignore this error.